Headlines/src/Controllers/SettingsViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SettingsViewController.swift
+//  Canillitapp
+//
+//  Created by Ezequiel Becerra on 25/01/2018.
+//  Copyright © 2018 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+import UserNotifications
+
+class SettingsViewController: UIViewController {
+    @IBOutlet weak var subscribeButton: UIButton!
+
+    private func registerForRemoteNotifications() {
+        UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound]) { (granted, error) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/2116f8d687ff9cf2ddfe00a13535cdb270d1c54c/Headlines/src/Controllers/SettingsViewController.swift#L16"></span>Unused parameter “error” in a closure should be replaced with _.</td>
    </tr>
  </tbody>
</table>